pkg/gofr/gofr.go

```diff
@@ -93,9 +94,52 @@ func New() *App {
 
 	app.subscriptionManager = newSubscriptionManager(app.container)
 
+	// static fileserver
+	currentWd, _ := os.Getwd()
+	checkDirectory := fmt.Sprintf("%s%spublic", currentWd, getFilePathSplitter())
+
+	if _, err := os.Stat(checkDirectory); !os.IsNotExist(err) {
+		app.AddStaticFiles("public", checkDirectory)
+	}
+
 	return app
 }
 
+func getFilePathSplitter() string {
+	fileSplitter := ""
+
+	switch runtime.GOOS {
+	case "windows":
+		fileSplitter = "\\"
+	case "linux":
+	case "darwin":
+	case "android":
+	case "ios":
+		fileSplitter = "/"
+	}
+	return fileSplitter
+}
+
+func (a *App) AddStaticFiles(endpoint, filePath string) {
+	a.httpRegistered = true
+	dupFilePath := ""
+	fileSplitter := getFilePathSplitter()
+	if filePath[:2] == "./" {
+		dupFilePath, _ = os.Getwd()
+		dupFilePath += fileSplitter + filePath + fileSplitter
```

Comment:
fileSpliter and getFilePathSplitter seems to me to reimplement filepath.Join

https://pkg.go.dev/path/filepath#Join

So I would discourage using the home made thing you add here

Reply:
Thanks @ccoVeille, I will make the changes and push the update in few minutes.